src/GovUk.Education.ExploreEducationStatistics.Content.Model/Extensions/ReleaseAmendmentExtensions.cs

```diff
@@ -27,9 +27,18 @@ public static Release CreateAmendment(this Release release, DateTime createdDate
 
             var context = new Release.CloneContext(amendment);
 
+            // Copy ReleaseContentSections/ContentSections and ContentBlocks associated with those sections
             amendment.Content = amendment
                 .Content
-                .Select(rcs => rcs.Clone(context))
+                .Select(rcs => rcs.Clone(context)) // Old/New ContentBlock pairs added to context here
+                .ToList();
+
+            // Copy ReleaseContentBlocks/ContentBlocks not associated with any ContentSection
+            // ReleaseContentBlocks only contains DataBlocks - this can be cleaned up in TODO: EES-1568
+            // DataBlocks copied previously are fetched from context so not cloned twice
+            amendment.ContentBlocks = amendment
+                .ContentBlocks
+                .Select(rcb => rcb.Clone(context)) // Old/New ContentBlock pairs added to context here
```

Comment:
One of these comments saying `ContentBlock pairs` I'm guessing needs a slight tweak.

Reply:
I intended them to be identical. I wanted to highlight that these lines were adding to `context` - rightly or wrongly, I didn't expect `Clone` to be changing the `context`, and in order to discover that it does, you need to dig down all the way to `ContentBlock#Clone(context)`.

I was tempted to bring the cloning code into `ReleaseAmendmentExtensions`. The code is specific to the creation of amendments - someone who stumbled upon it in `ReleaseContentBlock`, `ReleaseContentSection`, or the other `Clone(context)` methods  might get confused - and I think it would make this section of code in `CreateAmendment` easier to follow.

Reply:
Yeah that's not a bad idea